xmscore/python/misc/PyUtils.cpp

```diff
@@ -66,7 +98,7 @@ boost::shared_ptr<VecPt3d> VecPt3dFromPyIter(const py::iterable& pts)
   boost::shared_ptr<xms::VecPt3d> vec_pts(new xms::VecPt3d());
   for (auto item : pts) {
     if(!py::isinstance<py::iterable>(item)) {
-      throw py::type_error("First arg must be a n-tuple of 3-tuples");
+      throw py::type_error("First arg must be a n-tuple of 0, 1, 2, or 3-tuples");
```

Comment:
This isn't actually accurate, since we are really just checking if it is an iterable. We should probably change either the error message or the check.